data-model/SDP-sample-records/person_sample2.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/person/3",
+  "label": "Basil of Caesarea (approximately 329-379)",
+  "entity_type": "person",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379",
+  "description": "Greek patristic author and Bishop of Caesarea in Cappadocia.",
+  "authority_files": [{
+      "id": "https://w3id.org/haf/person/641569108508",
+      "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379"
+    },
+    {
+      "id": "https://viaf.org/viaf/88967224",
+      "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379"
+    },
+    {
+      "id": "http://syriaca.org/person/376",
+      "uniform_name": "Basil of Caesarea"
+    }
+  ],
+  "associated_dates":
+  [
+    {
+      "type": "birth",
+      "date_human_readable": "approximately 329",
+      "date_normalized": "0320/340"
+    },
+    {
+      "type": "death",
+      "date_human-readable": "approximately 379",
+      "date_normalized": "0370/0390"
+    }
+  ],
+  "associated_entities": [{
+      "type": "place",
+      "relation": "bishop-of",
+      "id": "https://sinaimanuscripts.librar.ucla.edu/place/100",
+      "notes": []
+    },
+    {
+      "type": "work",
```

Comment:
this would likely be a backlink/computed field from the work record rather than specified in the author record